File: tests/ClassMembers.cs
namespace Demo {
    public partial class Sample {
        // TODO: field count: int -> declare a field
        public int count;
        public string Name { get => get_Name; set => set_Name = value; }
        // TODO: const DefaultCount -> define a constant
        public const int DefaultCount = 10;
        public void Create() {
            base.Create();
        }
        public void Destroy() {
            base.Destroy();
        }
    }
}

File: tests/ClassMembers.pas
namespace Demo;

type
  Sample = public class
  public
    count: Integer;
    property Name: String read get_Name write set_Name;
    const DefaultCount : Integer = 10;
    constructor Create;
    destructor Destroy;
  end;

implementation

constructor Sample.Create;
begin
  inherited;
end;

destructor Sample.Destroy;
begin
  inherited;
end;
